whisk/client.go

```diff
@@ -652,10 +652,16 @@ func IsHttpRespSuccess(r *http.Response) bool {
 func IsResponseResultSuccess(data []byte) bool {
 	errResp := new(WhiskErrorResponse)
 	err := json.Unmarshal(data, &errResp)
-	if err == nil && errResp.Response != nil {
+
+	if err != nil {
+		Debug(DbgWarn, "IsResponseResultSuccess: failed to parse response result: %v\n", err)
+		return false
+	}
+
+	if errResp.Response != nil {
 		return errResp.Response.Success
 	}
-	Debug(DbgWarn, "IsResponseResultSuccess: failed to parse response result: %v\n", err)
+
```

Comment:
let's see...   the affected method is only looking for a whisk response (i.e. correct unmarshalled json with a "result" object).. the warning message is provided when that condition is not met.   so maybe at this spot the message should be 
```Debug(DbgWarn, "IsResponseResultSuccess: http response did not contain a 'response' field; assuming successful response\n")```  ???